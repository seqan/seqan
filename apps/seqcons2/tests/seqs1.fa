>simulated.1
GTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACC
>simulated.2
CGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCACCGGCCCGGCTAACAGC
>simulated.3
TCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCAACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATAT
>simulated.4
TACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGCGGCTCAGGCGTGGATTGCCTTGACTTCCGTAGCTTGGGGGGTAGCCGGCA
>simulated.5
AAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGA
>simulated.6
TGCCAGAAGTGTATGGCACTCCCGCTGTTAGCCGGGCCGTTGCCGGCTACCCCCCAAGCTGCGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTC
>simulated.7
GTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTATGATCCAGTAGGTCCACGGAATCAAGGC
>simulated.8
CCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGAATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATA
>simulated.9
CGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATATAGTCTGTAAGCGCAAGGGATAGGGCCGACACACC
>simulated.10
CCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTAG
>simulated.11
TTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTAT
>simulated.12
GCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCCTTA
>simulated.13
CGTAAGTGATTTCGTAGCGAACCTACCCAGCTCTAAGGGTCGGTCTGATCCAGTAGGTCCACGGAATCAAGGCGGCTCATGCGTGGATTGCCTTGACTTC
>simulated.14
TTGCGCTTACAGACTTTATTTCTTCGTGAGGGAGTTGACCCATGCAGAACCAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCACCAG
>simulated.15
GCACTCCCGCTGTTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCGGCCTTGATTCCGTGGACCTACTGGA
>simulated.16
GAGTTGACCCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAA
>simulated.17
CCATGCAGAACGAGAGCTTGCCGCTGAATGGGAACATCGCCGACGTTAGAGATGGCCCCAGACAGGCATATCGTAAGTGATTTCGTAGCGAACCTACCCA
>simulated.18
CCTACTGGATCATACCGACCCTTAGAGCTGGGTAGGTTCGCTACGAAATCACTTACGATATGCCTGTCTGGGGCCATCTCTAACGTCGGCGATTTTCCCA
>simulated.19
CTCTAACGTCGGCGATGTTCCCATTCAGCGGCAAGCTCTCGTTCTGCATGGGTCAACTCCCTCACGAAGAAATAAAGTCTGTAAGCGCAAGGGATAGGGC
>simulated.20
TTAGCCGGGCCGGTGCCGGCTACCCCCCAAGCTACGGAAGTCAAGGCAATCCACGCATGAGCCGCCTTGATTCCGTGGACCTACTGGATCATACCGACCC
