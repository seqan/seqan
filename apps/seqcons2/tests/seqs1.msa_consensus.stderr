seqcons2: the given value 'msa_consensus' is not in the list of allowed values [nop, realign, align_consensus, overlap_consensus, contig_consensus, pos_consensus]
